_includes/docs_menu.html

```diff
@@ -13,6 +13,7 @@
     <li><a href="{{ site.baseurl }}/docs/localize">Localize Your Connector</a></li>
     <li><a href="{{ site.baseurl }}/docs/share">How to Run Your "Under Development" Connector</a></li>
     <li><a href="{{ site.baseurl }}/docs/package-sign">How to Package and Sign Your Connector for Distribution</a></li>
+    <li><a href="{{ site.baseurl }}/docs/run-taco">How to Run your Packaged Connector (Taco)</a></li>
```

Comment:
Let's remove "How to" from each of the three headings.

Reply:
Done.